src/robusta/core/sinks/sink_factory.py

```diff
@@ -21,12 +21,12 @@
 class SinkFactory:
     @classmethod
     def create_sink(
-            cls, sink_config: SinkConfigBase, account_id: str, cluster_name: str, signing_key: str
+            cls, sink_config: SinkConfigBase, account_id: str, cluster_name: str, signing_key: str, registry
```

Comment:
Since account_id, cluster_name, signing_key are all taken from registry.global_config, do you think it will be cleaner to just pass registry, and store it on the SinkBase.
Then, each concrete sink can use whatever params it needs from registry.global_config
WDYT?